### ISSUE TITLE
Error in scvi.model.AmortizedLDA.load

### ISSUE THREAD

**Issue by nilesh-iiita** [scverse/scvi-tools#1646]
When trying to load saved model using 'scvi.model.AmortizedLDA.load' it again starts training the model. 

```python
model_loaded = scvi.model.AmortizedLDA.load("SCC_Topic_model/", adata=adata, use_gpu=True)
```

```pytb
INFO     File SCC_Topic_model/model.pt already downloaded                                    
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
/home/nileshkr/.conda/envs/Hackin_Omics/lib/python3.10/site-packages/pytorch_lightning/trainer/configuration_validator.py:131: UserWarning: You passed in a `val_dataloader` but have no `validation_step`. Skipping val loop.
  rank_zero_warn("You passed in a `val_dataloader` but have no `validation_step`. Skipping val loop.")
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
```

#### Versions:

> '0.17.1'

**Comment by adamgayoso** [comment]
This is fixed by #1636

**Comment by adamgayoso** [comment]
For now you can ctrl+c while it appears to be training, and then it will stop and load the correct weights
